app/backend/src/couchers/jobs/handlers.py

```diff
@@ -57,6 +58,54 @@ def process_purge_login_tokens(payload):
         )
 
 
+def process_generate_message_notifications(payload):
+    """
+    Generates notifications for a message sent to a group chat
+    """
+    logger.info(f"Sending out notifications for message_id = {payload.message_id}")
+
+    with session_scope() as session:
+        message, group_chat = session.execute(
+            select(Message, GroupChat)
+            .join(GroupChat, GroupChat.conversation_id == Message.conversation_id)
+            .where(Message.id == payload.message_id)
+        ).one()
+
+        if message.message_type != MessageType.text:
+            logger.info(f"Not a text message, not notifying. message_id = {payload.message_id}")
+            return
+
+        notify_args = {
+            "topic": "chat",
+            "key": str(message.conversation_id),
+            "action": "message",
+            "icon": "message",
+            "title": f"{message.author.name} sent a message in {group_chat.title}.",
+            "content": message.text,
+            "link": urls.chat_link(chat_id=message.conversation_id),
+        }
```

Comment:
Avoid the intermediary dict and pass the parameters in the call directly.

Reply:
Fixed